lib/kino/table.ex

```diff
@@ -31,6 +31,10 @@ defmodule Kino.Table do
           fields: list(%{term() => String.t()})
         }
 
+  @type row_based :: list(row())
+
+  @type columnar :: %{term() => list(String.t())}
```

Comment:
Why is it `String.t()`? Isn't it anything encodable to json?

Reply:
We encode all values to strings on Elixir side already, this matches `fields:` above :)